devices/atlantic.js

```diff
@@ -0,0 +1,127 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const utils = require('../lib/utils');
+const assert = require('assert');
+const e = exposes.presets;
+const ea = exposes.access;
+
+
+const thermostatPositions = {
+    'barley_closed': 0,
+    'quarter_open': 1,
+    'half_open': 2,
+    'three_quarters_open': 3,
+    'fully_open': 4,
+    'swing': 5,
+};
+
+const energyMode = {
+    'eco': 0,
+    'normal': 1,
+    'powerfull': 2,
+};
+
+const tzLocal = {
+    quiet_fan: {
+        key: ['quiet_fan'],
+        convertSet: async (entity, key, value, meta) => {
+            assert(typeof value === 'boolean');
+            // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+            await entity.write('hvacFanCtrl', {0x1000: {value: 1, type: 0x10}});
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === 5) {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 1, type: 0x10}});
+            } else {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 0, type: 0x10}});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4273: {value: index, type: 0x04}});
+            }
+            return {state: {ac_louver_position: value}};
+        },
+    },
+    energy_mode: {
+        key: ['energy_mode'],
+        convertSet: async (entity, key, value, meta) => {
```

Comment:
I have two options: `powerful` and `eco` mode. I decided that they will be mutually exclusive, and I added a `normal` mode when both `eco` and `powerful` are off.

Reply:
What is your question here?